docs/theming/high-contrast-mode.md

```diff
@@ -0,0 +1,228 @@
+---
+title: Hide Contrast Mode
+initialTab: 'preview'
+inlineHtmlPreviews: true
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<head>
+  <title>High Contrast Mode to Increase Color Contrast</title>
+  <meta
+    name="description"
+    content="Developers are adding high contrast mode CSS on native applications to support their user preferences. Read to learn more about high contrast color schemes for Ionic apps."
+  />
+</head>
+
+Ionic ships with themes for increased contrast for users with low vision. These themes work by increasing the contrast between foreground content, such as text, and background content, such as UI components. Ionic provides light and dark variants for high contrast.
+
+## Overview
+
+The [Ionic colors](./colors) in the high contrast theme have been updated to meet [Level AAA color contrast](https://www.w3.org/WAI/WCAG21/Understanding/contrast-enhanced.html) as defined by the Web Content Accessibility Guidelines (WCAG) when used with the appropriate contrast color. The theme also seeks to improve the contrast of UI components such as the border on an Item. However, Ionic's high contrast theme prioritizes text. This means that if increasing the contrast of a UI component against the page background would significantly reduce the contrast of the component's text against the component's background, the UI component contrast will not be modified.
```

Comment:
This is not super clear, so open to ideas for how to better explain this

Reply:
Courtesy of Chat GPT:

"The Ionic high contrast theme has been enhanced to adhere to Level AAA color contrast standards as outlined in the Web Content Accessibility Guidelines (WCAG), particularly when paired with the appropriate contrast color. Notably, improvements have been made to the contrast of UI components, including item borders. However, it's important to note that within the high contrast theme, priority is given to text legibility. Consequently, if adjusting the contrast of a UI component against the background would significantly compromise the contrast between the component's text and its background, the contrast of the UI component will remain unchanged."

Reply:
Tried to combine in https://github.com/ionic-team/ionic-docs/pull/3431/commits/851d6f79ef3e8936af401bb396a28bc7e8625f0c. Lmk your thoughts